Common/MLKeychain.swift

```diff
@@ -40,7 +40,7 @@ public class MLKeychain {
         let keychainQuery: [String: Any] = [
             kSecClassValue: kSecClassGenericPasswordValue,
             kSecAttrAccountValue: key,
-            kSecReturnDataValue: kCFBooleanTrue,
+            kSecReturnDataValue: kCFBooleanTrue!,
```

Comment:
Not sure why I have been told by Xcode to fix this one. It was working a year ago I suppose?

Reply:
I'll have to look into this some. I'd love to avoid any force unwraps.